script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
+else
+  export FLUTTER_CHANGED_PACKAGES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
```

Comment:
I missed the "also" part of your comment.

I am finding it difficult to imagine why you would do that in a single PR. Maybe you changed something globally and had to touch a subset of plugins to make it work? Still odd. I would definitely reject PRs that are not correlated (e.g. "I am updating SharedPreferences behavior and oh by the way, I also wanted to change the build script"). Those should be separate PRs. Thoughts?